tools/testdata-scripts/src/bin/download-repo-sources.rs

```diff
@@ -57,10 +58,26 @@ fn main() -> eyre::Result<()> {
 
     let output_path = &args.out;
 
-    let cached = cached_path::CacheBuilder::new()
-        .freshness_lifetime(u64::MAX)
-        .build()
-        .unwrap();
+    fn cached(resource: &str) -> Result<PathBuf, DataError> {
+        let root = std::env::var_os("ICU4X_SOURCE_CACHE")
+            .map(PathBuf::from)
+            .unwrap_or_else(|| std::env::temp_dir().join("icu4x-source-cache/"))
```

Comment:
Question: should we go back to using `dirs::cache_dir()`?

Reply:
What's the advantage? It's an extra dependency.

Reply:
I assume `cache_dir` is more likely to be persisted, so better for a "cache", whereas `/tmp` can be deleted at any time

Reply:
I think it's good if it gets cleared occasionally, with `cache_dir` we might get complaints that we're not cleaning up after ourselves :)